### DISCUSSION TITLE
Dịch Agglomerative clustering, BIRCH, Mean-Shift, Affinity propagation, Spectral clustering, GMM

### DISCUSSION THREAD

**Discussion by rootonchair** [mlbvn/handson-ml2-vn#66] [Terminology] [answered]
|Từ gốc| Tạm dịch | Thống nhất
|------|------|------|
|Agglomerative clustering| *giữ nguyên*| |
|BIRCH (Balanced Iterative Reducing and Clustering using Hierarchies)| ~~Lặp Cân bằng Giảm thiểu và Phân cụm bằng phương pháp Phân cấp~~ | *giữ nguyên* |
|Mean-Shift| Dịch-Trung bình | |
|Affinity propagation| *giữ nguyên* | |
|Spectral clustering| Phân cụm Phổ | |
|Gaussian Mixture Model (GMM)| ~~Mô hình Gauss hỗn hợp~~ | Mô hình hỗn hợp Gauss |

@mlbvn/handson-ml2

**Comment by cuongvng** [comment]
BIRCH mình đọc tên thì hiểu là cả "Reducing and Clustering" sẽ đi kèm với "Balanced Iterative " và "using Hierarchies".
Nhưng đọc qua thuật toán thì thấy đây là về clustering, chưa hiểu "Reduce" cái gì?

**Reply by rootonchair** [reply]
Em thấy reduce ám chỉ việc nó chỉ giữ thông tin về dataset đủ để thực hiện cluster

**Reply by cuongvng** [reply]
`Giảm thiểu và Phân cụm Lặp Cân bằng sử dụng Phân cấp`?
Khó nhỉ, TV danh từ đứng trước tính từ, cơ mà 2 gerunds dịch ra thế sẽ thành động từ "Giảm thiểu và Phân cụm" rồi nên "Lặp Cân bằng" lại bị nhầm sang danh từ, không phải tính từ nữa.

Hay là giữ nguyên?

**Reply by rootonchair** [reply]
Giữ nguyên cũng ok, vì từ này không dùng nhiều ạ

**Comment by minhduc0711** [comment]
* `Agglomerative clustering`: từ agglomertive mình tra từ điển có nghĩa là `tích tụ`, nghe có vẻ khớp với ý tưởng `bottom-up`
* `Affinity propagation`: mình thấy có vài nguồn dịch là `lan truyền độ tương tự`, nhưng cũng chưa chắc lắm vì mình chưa xem kỹ thuật toán
* `Gaussian Mixture Model`: đảo thành `mô hình hỗn hợp Gauss` được không nhỉ, vì `mixture model` là 1 cụm đi với nhau

**Reply by ngcthuong** [reply]
- Vậy dịch là Agglomerative clustering là "tích cụm"?

Từ affinity ở đoạn nào em nhỉ?

**Reply by cuongvng** [reply]
![image](https://user-images.githubusercontent.com/30251584/113567230-6ca4b280-9638-11eb-8ff4-adca7809084f.png)

yeah em thấy `tích cụm` ổn

**Reply by minhduc0711** [reply]
@ngcthuong đây ạ
> Affinity propagation::
This algorithm uses a voting system, where instances vote for similar instances to be their representatives, and once the algorithm converges, each representative and its voters form a cluster. Affinity propagation can detect any number of clusters of different sizes. Unfortunately, this algorithm has a computational complexity of O(_m_^2^), so it too is not suited for large datasets.

**Reply by rootonchair** [reply]
@minhduc0711 đã update đối với GMM

**Comment by lkhphuc** [comment]
- `Affinity propagation` mình tạm dịch là `lan truyền ái lực` nhé. Từ này chỉ xuất hiện một lần nên chỉ cần tạm dịch thôi.

**Comment by minhduc0711** [comment, marked as answer]
chốt như này nhé @rootonchair 
|Từ gốc| Thống nhất
|------|------|
|Agglomerative clustering| tích cụm |
|BIRCH (Balanced Iterative Reducing and Clustering using Hierarchies) |  *giữ nguyên* |
|Mean-Shift| Dịch-Trung bình |
|Affinity propagation| lan truyền ái lực |
|Spectral clustering| Phân cụm Phổ | 
|Gaussian Mixture Model (GMM)| Mô hình hỗn hợp Gauss |
